docker-compose.yml

```diff
@@ -1,12 +1,16 @@
 # This is a test docker-compose file that is not intended for production use!
 name: recall
 
+networks:
+  recall-network:
+    driver: bridge
+
 services:
   db:
     image: postgres
     restart: always
-    ports:
-      - "5432:5432"
+    networks:
+      - recall-network
```

Comment:
Can/should we leave the DB accessible on the host? Has been very helpful for DB inspection from the host.